plugin.py

```diff
@@ -79,27 +144,191 @@ class TellRecord(Base):
 # session.rollback()
 session = DBSession()
 
+
 class Tell(callbacks.Plugin):
     """MemoServ replacement with extra features."""
     threaded = True
 
+    queryTell = PostTell()
```

Comment:
This naming seems weird. Maybe change `PostTell` to `TellLib` and then `queryTell` -> `tellLib`?

Reply:
Post as in after the fact.. My thoughts.